tests/testthat/test-type-convert.R

```diff
@@ -6,3 +6,8 @@ test_that("missing values removed before guessing col type", {
 
   expect_equal(df2$x, c(NA, 10L))
 })
+
+test_that("requires data.frame input", {
+  not_df <- matrix(letters[1:4], nrow = 2)
+  expect_error(type_convert(not_df))
```

Comment:
For future reference, I think it's bad practice to use `expect_error()` without a string because (e.g) `expect_error(type_c0nvert(not_df))`